modules/util/source-formatter/src/main/java/com/liferay/source/formatter/upgrade/GradleBuildFile.java

```diff
@@ -114,7 +114,8 @@ public String getSource() {
 	}
 
 	public List<String> getSourceLines() {
-		return Arrays.asList(_source.split(System.lineSeparator()));
+		return ListUtil.copy(
```

Comment:
@slnn Please try to use ListUtil.fromArray

Reply:
@slnn Maybe `ListUtil.fromString` can also do the work, but we may need to double check whether it works in Windows.